.github/workflows/semver-checks-pr-label.yml

```
@@ -73,7 +73,7 @@ jobs:
               owner: context.repo.owner,
               repo: context.repo.repo,
               issue_number: issue_number,
-              body: ':rotating_light: API BREAKING CHANGE DETECTED'
+              body: ':rotating_light: API BREAKING CHANGE DETECTED\n\nTo see the changes click details on "Check semver breaks / PR Semver - stable toolchain" job then expand "Run semver checker script" and scroll to the end of the section.'
```

We could also do:
```suggestion
              body: ':rotating_light: API BREAKING CHANGE DETECTED\n\nTo see the changes browse the recent actions output at <https://github.com/rust-bitcoin/rust-bitcoin/actions/workflows/semver-checks.yml> or in the "Checks" PR tab'
```

I am not certain of the `<URL>` using `<>` will work though...

---

That points to the yml, we need URL to specific run. I'd rather not spend time messing with it now that we have a broken PR we're unable to debug because this change isn't in.